scripts/ci-build-azure-ccm.sh

```diff
@@ -28,6 +28,11 @@ source "${REPO_ROOT}/hack/ensure-go.sh"
 # shellcheck source=hack/parse-prow-creds.sh
 source "${REPO_ROOT}/hack/parse-prow-creds.sh"
 
+: "${AZURE_STORAGE_ACCOUNT:?Environment variable empty or not defined.}"
```

Comment:
why is this needed? I don't see it getting used later in the script

Reply:
nevermind I see now it's used to upload the binaries to a storage blob, same usage as `ci-build-kubernetes.sh`